Lib/uuid.py

```diff
@@ -756,6 +757,71 @@ def uuid5(namespace, name):
     int_uuid_5 |= _RFC_4122_VERSION_5_FLAGS
     return UUID._from_int(int_uuid_5)
 
+_last_timestamp_v7 = None
+_last_counter_v7 = 0  # 42-bit counter
+
+def uuid7():
```

Comment:
Slip in some PEP 8 whitespace:
```suggestion

_last_timestamp_v7 = None
_last_counter_v7 = 0  # 42-bit counter


def uuid7():
```

Reply:
The file is not really PEP-8 friendly because some functions are separated by two empty lines, others aren't. I would prefer either having all `uuid` functions without PEP-8 or all of them with (it feels a bit weird that `uuid1` to `uuid4` won't have the double spacing but UUID 6, 7, and 8 would).

Reply:
Then add the rest too :) PEP 8: "although this is also an opportunity to clean up someone else’s mess (in true XP style)."

Reply:
I guess I can do it. The module is very very slow for updates so I don't think we'll have issues with backports :)